graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -100,9 +104,13 @@ def create_for_session(
 
     @staticmethod
     def _configure_aura(config: dict[str, Any]) -> None:
-        config["max_connection_lifetime"] = 60 * 8  # 8 minutes
-        config["keep_alive"] = True
-        config["max_connection_pool_size"] = 50
+        # defaults as documented in https://support.neo4j.com/s/article/1500001173021-How-to-handle-Session-Expired-Errors-while-connecting-to-Neo4j-Aura
+        config.setdefault("max_connection_lifetime", 60 * 50)  # 50 minutes
+        config.setdefault("keep_alive", True)
+        config.setdefault("max_connection_pool_size", 50)
```

Comment:
Should we also set these defaults when talking to aura db and sessions?

Reply:
`_configure_aura` is used for the auradb as well. its set in both right now